backend/tests/test_workers.py

```
@@ -970,10 +970,140 @@ async def test_analyze_backing_vocals_handles_missing_stems(self):
+
+        # Expected stem filenames
+        expected_stems = {
+            'vocals_clean': f"{base_name} (Vocals {clean_model}).flac",
+            'lead_vocals': f"{base_name} (Lead Vocals {backing_model}).flac",
+            'backing_vocals': f"{base_name} (Backing Vocals {backing_model}).flac",
+            'bass': f"{base_name} (Bass {other_model}).flac",
+            'drums': f"{base_name} (Drums {other_model}).flac",
+        }
+
+        # Verify naming convention
+        for key, name in expected_stems.items():
```

Ruff (0.14.10)

Loop control variable `key` not used within loop body

Rename unused `key` to `_key`

(B007)